result/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from account.models import Lecturer, Student
+from course.models import Courses
+from core.choices import *
+# Create your models here.
+
+
+
+
+
+class Result(models.Model):
+    student = models.OneToOneField(Student, on_delete=models.CASCADE, null=True, blank=True, related_name="results")
```

Comment:
This should be foreignkey as one student instance can have many result instances.